target/linux/ipq806x/patches-5.10/097-1-drivers-net-dsa-qca8k-add-initial-support-for-led-co.patch

```diff
@@ -0,0 +1,100 @@
+From 392e2713f263b1e5b8b602b2f6ecc84482302c33 Mon Sep 17 00:00:00 2001
+From: Ansuel Smith <ansuelsmth@gmail.com>
+Date: Thu, 4 Mar 2021 10:03:32 +0100
+Subject: [PATCH 09/10] drivers: net: dsa: qca8k: add initial support for led
+ config
+
+Some device use a non standard configuration for the LED blink freq.
+Add support for this specific configuration. If needed other LED
+configuration can be added later.
+
+Signed-off-by: Ansuel Smith <ansuelsmth@gmail.com>
+---
+ drivers/net/dsa/qca8k.c | 49 +++++++++++++++++++++++++++++++++++++++++
+ drivers/net/dsa/qca8k.h | 10 +++++++++
+ 2 files changed, 59 insertions(+)
+
+--- a/drivers/net/dsa/qca8k.c
++++ b/drivers/net/dsa/qca8k.c
+@@ -719,6 +719,51 @@ qca8k_setup_mac_pwr_sel(struct qca8k_pri
+ }
+ 
+ static int
++qca8k_setup_led_ctrl(struct qca8k_priv *priv)
++{
++	struct device_node *node = priv->dev->of_node;
++	u32 mask, val, reg;
```

Comment:
'mask' initialization is still missing...
I think that something like `mask = QCA8K_LED_BLINK_FREQ_MASK;` is required.